addons/website_customer/views/website_customer_templates.xml

```diff
@@ -196,7 +196,7 @@
                     <label class="cursor-pointer" t-if="industry_dict['industry_id'] and industry_dict['industry_id'][0] == current_industry_id and true or false" t-out="industry_dict['industry_id'][1]"/>
                 </t>
             </a>
-            <div class="dropdown-menu">
+            <div class="dropdown-menu" style="max-height: 200px; overflow-y: auto;">
```

Comment:
Why doing this with a class in the other app but with inline style here? Why not unifying everything? And in stable, for users to receive the fix, why not just add a CSS rule like "all dropdown menus in the filters area"?